src/components/GenreSelect/GenreSelect.css

```diff
@@ -0,0 +1,20 @@
+.genre-select {
+    height: 50px;
+    display: flex;
+    gap: 30px;
+    border-bottom: 3px solid #424242;
+}
+
+.genre-select-button {
+    color: white;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: left;
+}
+
+.selected-button {
+    margin-bottom: -3px;
+    border-bottom: 3px solid #F65261;
+}
```

Comment:
There is a problem with this styling approach - all the styles are global by default. It means that, if in any other component you also add `.selected-button` class, it is going to be applied to `GenreSelect` component, and vice versa. This may not be a problem when there are few components and 1 developer, but it quickly becomes a headache on bigger projects. 
I suggest that you look into ways of avoiding this problem - CSS Modules, css-in-js, CSS methodologies, etc. Pick the one that suits you best and stick to it.

Reply:
I first tried the styled-components library and, to be honest, I really didn't like this approach. I'll try other lib later maybe

Reply:
You can still achieve scoping without adding any external libraries - for example by using BEM naming convention or CSS modules: https://blog.logrocket.com/a-deep-dive-into-css-modules/